app/serializers/api/v1/order_serializer.rb

```diff
@@ -33,6 +35,41 @@ def unread_messages_count__sql
       "(select count(*) from subscriptions s where s.order_id = orders.id and s.state = 'unread' and s.user_id = orders.created_by_id)"
     end
 
+    def user_submitted_order_count
+      Order.users_order_count(object.created_by_id, 'submitted')
```

Comment:
Is it ok to request data from serializer? 
I think serializer should only be used for serializing data. 

CC: @namrataukirde